content/zh/news/support/announcing-1.19-eol/index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: 对 Istio 1.19 的支持于 2024 年 04 月 24 日结束
+subtitle: 支持公告
+description: Istio 1.19 生命周期即将结束的公告。
+publishdate: 2024-03-27
+---
+
+根据 Istio 的[支持政策](/zh/docs/releases/supported-releases#support-policy)，
+针对 1.19 这类次要版本的支持将持续到 N+2 次要版本（在本例中为 1.21）发布后的六周。
+[Istio 1.21 于 2024 年 03 月 13 日发布](/zh/news/releases/1.21.x/announcing-1.21/)，
+所以对 1.19 的支持于 2024 年 04 月 24 日结束。
```

Comment:
```suggestion
[Istio 1.21 于 2024 年 3 月 13 日发布](/zh/news/releases/1.21.x/announcing-1.21/)，
所以对 1.19 的支持于 2024 年 4 月 24 日结束。
```

Reply:
Hi @wilsonwu ,
Thanks for your comment. I have changed the data format.
Please let me know if you have any other questions.